l10n/configs/special-templates.toml

```diff
@@ -0,0 +1,16 @@
+# For locales with dedicated templates that don't go through
```

Comment:
I'm curious on the use case here - if content doesn't go through an L10n process, then we typically use hard-coded, locale-specific templates that bedrock already has functionality to support. Is this something that we'd need to make exceptions for here?

Reply:
Yes.